frontend/pages/admin/OrgSettingsPage/cards/Advanced/Advanced.tsx

```diff
@@ -92,6 +144,10 @@ const Advanced = ({
         host_expiry_enabled: enableHostExpiry,
         host_expiry_window: Number(hostExpiryWindow),
       },
+      activity_expiry_settings: {
+        activity_expiry_enabled: deleteActivities,
+        activity_expiry_window: Number(activityExpiryWindow),
```

Comment:
Do we need the conversions here and for `host_expiry_window` since they should already be `number`s?

Reply:
Also for `port` above, `Number(undefined)` gives `NaN`, maybe there's a more explicit way we can say "nothing", maybe `appConfig.smtp_settings?.port || null`?

Reply:
Changing Port, host expiry window and activity expiry window to all default to null